packages/arb-token-bridge-ui/src/pages/_app.tsx

```diff
@@ -1,7 +1,7 @@
 import type { AppProps } from 'next/app'
 import Head from 'next/head'
 import * as Sentry from '@sentry/react'
-import { BrowserTracing } from '@sentry/tracing'
+import { BrowserTracing } from '@sentry/browser'
```

Comment:
As part of this PR, we might as well bump Sentry versions to latest?

Reply:
Updated